pypesto/objective/priors.py

```diff
@@ -560,3 +614,62 @@ def function(x):
         return constant
 
     return function
+
+
+def _sample_from_prior(
+    prior_type: str,
+    prior_parameters: list,
+    n_samples: int,
+    rng: np.random.Generator,
+) -> np.ndarray:
+    """
+    Sample from individual prior distributions.
+
+    Parameters
+    ----------
+    prior_type:
+        Type of the prior distribution.
+    prior_parameters:
+        Parameters of the prior distribution.
+    n_samples:
+        Number of samples to draw.
+    rng:
+        Random number generator.
+
+    Returns
+    -------
+    samples:
+        Array of samples from the prior distribution.
+    """
+    if prior_type in [C.UNIFORM, C.PARAMETER_SCALE_UNIFORM]:
+        low = prior_parameters[0]
+        high = prior_parameters[1]
+        return rng.uniform(low, high, n_samples)
+
+    elif prior_type in [C.NORMAL, C.PARAMETER_SCALE_NORMAL]:
+        mean = prior_parameters[0]
+        std = prior_parameters[1]
+        return rng.normal(mean, std, n_samples)
```

Comment:
Where is this transformed into a sample from the log-normal distribution, if parameter scale is log?